lib/Models/Setting.php

```diff
@@ -26,6 +26,7 @@ function __construct() {
    * @deprecated
    */
   public static function getValue($key, $default = null) {
+    @trigger_error('Calling Settings::getValue() is deprecated and will be removed.', E_USER_DEPRECATED);
```

Comment:
Why the `@`?

Reply:
I removed it. I added it when I noticed that it is used in Symfony but jumped to another issue and forgot research why... Seems that their approach is to always silence deprecation warnings and dev should opt in to get them.